src/SWP/Bundle/ContentBundle/Tests/Functional/Controller/ContentPushControllerTest.php

```diff
@@ -27,6 +27,9 @@ class ContentPushControllerTest extends WebTestCase
ff77e2556fde13bf9f383a33795760c42e693a40d0479794febf4.jpg"}, "baseImage": {"width": 1400, "mimetype": "image/jpeg", "poi": {"x": 840, "y": 401}, "media": "20170111140132/828ca0e06e013797aa2f32be119803f37843501c7618ea364b6b393f17e69708.jpg", "height": 933, "href": "https://s3.superdesk.org/superdesk-test-eu-west-1.s3-eu-west-1.amazonaws.com/20170111140132/828ca0e06e013797aa2f32be119803f37843501c7618ea364b6b393f17e69708.jpg"}, "original": {"width": 2048, "mimetype": "image/jpeg", "poi": {"x": 1228, "y": 586}, "media": "20170111140132/979ff3c8a001d6cb2a7071eab9be852211853990f8d60e693e38f79e972772ea.jpg", "height": 1365, "href": "https://s3.superdesk.org/superdesk-test-eu-west-1.s3-eu-west-1.amazonaws.com/20170111140132/979ff3c8a001d6cb2a7071eab9be852211853990f8d60e693e38f79e972772ea.jpg"}, "600x300": {"poi": {"x": 573, "y": 371}, "width": 598, "media": "sdsite/20170207070248/206d13a7-42aa-4618-b534-d56468a03e15.jpg", "height": 300, "href": "https://s3.superdesk.org/superdesk-test-eu-west-1.s3-eu-west-1.amazonaws.com/sdsite/20170207070248/206d13a7-42aa-4618-b534-d56468a03e15.jpg"}, "777x600": {"poi": {"x": 1184, "y": 586}, "width": 668, "media": "sdsite/20170207070248/49521a16-9b51-4ddd-a01a-59aa4db3e71f.jpg", "height": 517, "href": "https://s3.superdesk.org/superdesk-test-eu-west-1.s3-eu-west-1.amazonaws.com/sdsite/20170207070248/49521a16-9b51-4ddd-a01a-59aa4db3e71f.jpg"}}, "place": [], "pubstatus": "usable", "slugline": "gradac", "firstcreated": "2017-01-11T14:32:58+0000", "mimetype": "image/jpeg", "service": [{"code": "news", "name": "News"}], "byline": "Ljub. Z. Rankovi\u0107", "urgency": 3, "language": "en", "headline": "Smoke on the water", "versioncreated": "2017-01-11T14:52:05+0000", "description_text": "Smoke on the water on River Gradac corrected", "guid": "tag:localhost:2017:4bea4f26-d5a1-446b-8953-3096c0ad0f09", "body_text": "Gradac corrected", "version": "5"}}, "place": [], "firstcreated": "2017-02-07T07:46:48+0000", "body_html": "<p>some text and after that we should get image</p>\n<!-- EMBED START Image {id: \"embedded9582903151\"} -->\n<figure><img src=\"https://s3.superdesk.org/superdesk-test-eu-west-1.s3-eu-west-1.amazonaws.com/20161206171212/896bd89c2eaa29bbb8a953787a86615c5a9aaf16b4ad93b4ac7f7af23f0c459c.jpg\" alt=\"Bell Peppers\" srcset=\"https://s3.superdesk.org/superdesk-test-eu-west-1.s3-eu-west-1.amazonaws.com/20161206171212/896bd89c2eaa29bbb8a953787a86615c5a9aaf16b4ad93b4ac7f7af23f0c459c.jpg 450w, https://s3.superdesk.org/superdesk-test-eu-west-1.s3-eu-west-1.amazonaws.com/20161206171212/2577b421dbaf12be0af28be026e4fc4e5484b42aa745c35f071bb3da56e0aadc.jpg 777w\" /><figcaption>Few of a kind</figcaption></figure>\n<!-- EMBED END Image {id: \"embedded9582903151\"} -->\n<p>and after image again some text</p><p>footer content</p>", "service": [{"code": "news", "name": "News"}], "description_text": "abstract", "urgency": 3, "language": "en", "headline": "headline", "byline": "ADmin", "versioncreated": "2017-02-08T07:49:48+0000"}';
```

Comment:
this was passing before because we added feature to rely on `evolvedfrom` instead of a slug, `TEST_FEATUREMEDIA_ITEM_CORRECTED` const's value should have `evolvedfrom` property set to `urn:newsml:localhost:2017-02-07T07:46:48.027116:2cde1d3f-302f-4cf9-a4b9-809d2320cc00` (from `TEST_FEATUREMEDIA_ITEM`). This will reflect the whole workflow of correction properly.

There also should be additional test case when you push the same item (same guid) twice as you have done this now.